lambdas/db-migration/src/migrations/20210211172119_create_granules_executions_table.ts

```diff
@@ -0,0 +1,24 @@
+import * as Knex from 'knex';
+
+export const up = async (knex: Knex): Promise<void> =>
+  knex.schema.createTable('granules_executions', (table) => {
+    table
+      .integer('granule_cumulus_id')
+      .notNullable();
+    table.foreign('granule_cumulus_id')
+      .references('cumulus_id')
+      .inTable('granules')
+      .onDelete('CASCADE');
```

Comment:
:100: this is *far* better than trying to do this in code.   

We should make sure we're validating this in units..... which we are.  Great work.